docs/tutorials/vagrant-installOs.rst

```diff
@@ -0,0 +1,150 @@
+Perform an Unattended OS Install
+=================================
+
+Summary
+------------
+
+In this section, you will leverage RackHD to perform an unattended OS install onto a bare metal server.
+
+In this section, the Photon OS is used as the example. It is a minimal Linux container host, optimized to run on VMware platforms. Photon was chosen because it is small enough for the OS installation to take approximately 8 or 9 minutes. The same process that is used in this lab can be used to install other mainstream operating systems, such as Ubuntu, CentOS, and ESXi, which take longer to install.
+
+Install OS
+-----------------
+Before the following operations are excuted, you need to ssh the vagrant box first. ``vagrant ssh dev`` can enter the demo environment. ``vagrant ssh dev_ansible`` can enter development environment.
+
+Prerequisite
+~~~~~~~~~~~~~
+
+1. The quanta_d51 vnode which runs in vagrant is used as the OS-Install target node in this Lab. Get the information of vnode.
+
+.. code::
+
+  vagrant@rackhd:~$ curl http://localhost:8080/api/2.0/catalogs | jq '.' | grep D51 -B8 | grep node
+
+.. image:: ../_static/node_info.png
+     :align: center
+   
+    
+2. Use the mouse to select and copy ``node-id`` text( as shown in the below example snapshot). This ID will be used in the following steps.
```

Comment:
sometime 'node_id' sometime 'Node_ID' , 'node-id' ,please be aligned in all files